### PR TITLE
Update track configuration

### PR DESCRIPTION
**Pull request by wenngle**
- Write docs, including the about, installation, resources, and tests
  pages  (#35).
- Configure the online editor (#8).
- Add language blurb (#24).
- Add tags (#10).

I've also written a hello world exercise, but I think that should probably be a separate PR. I'm interested in working more on this track. What are the current plans for Nix on Exercism?

### CODE REVIEW

**Comment by terlar** [comment]
Thank you for picking this up. I’ve been meaning to do it myself, but life has been hectic. If we have decided on a way, I can help write exercises as well.

I was also going to suggest `lib.debug.runTests`. I was also wondering if flakes should be used to manage the nixpkgs input, but since it is still a experimental and there is no native integration for nix flakes and `lib.debug.runTests` I guess it might be worth holding off on that.

**Comment by wenngle** [comment]
I agree with you. Flakes aren't ready. They also don't fit our usecase well because we don't depend on external inputs besides nixpkgs and they add a lot of boilerplate code. 

For testing, `runTests` does most of the work, but it's lacking a good UX. In the future, I think it would be great if we could automatically generate test cases from the problem specification, but let's not get ahead of ourselves. Here's what I've hacked together for the hello world exercise:

`hello-world-test.nix`
```nix
with builtins; let
  lib = import <nixpkgs/lib>;
  helloWorld = import ./hello-world.nix;
  results = lib.debug.runTests {
    testHello = {
      expr = helloWorld.hello {};
      expected = "Hello, World!";
    };
  };
in
  if results == []
  then "All tests passed!\n"
  else
    deepSeq
    (map
      (t:
        trace ''
          ${t.name}:
            expected: ${t.expected}
            result: ${t.result}''
        t)
      results)
    (throw "${toString (length results)} tests failed!\n")
```

`hello-world.nix`
```nix
{
  hello = {}: "Goodbye, Mars!";
}
```

Anyway, the testing details can be worked out later. It's not critical for this PR.

File: docs/TESTS.md
Comment (ErikSchierboom) [inline docs/TESTS.md:9]: I recently learned that you can also use `shell` as the code block's type

```suggestion
```shell
```

File: docs/TESTS.md
Comment (ErikSchierboom) [inline docs/TESTS.md:13]:
```suggestion
```shell
```

**Review by ErikSchierboom — APPROVED** [review]
Two tiny nits. Let me know if you want to update the PR or not.

File: docs/INSTALLATION.md
Comment (ErikSchierboom) [inline docs/INSTALLATION.md:4]: As there are no additional level two headings, I'd probably just remove this.

**Comment by wenngle** [comment]
I made a few small changes in the Github web editor. I’ll squash the commits when I’m at my computer later.

**Comment by wenngle** [comment]
Thanks for the feedback. Please take another look at the changes before merging.

**Review by ErikSchierboom — APPROVED** [review]
Awesome!

**Pull request merged**
